lib/internal/process/pre_execution.js

```diff
@@ -128,9 +128,26 @@ function prepareExecution(options) {
 function setupSymbolDisposePolyfill() {
   // TODO(MoLow): Remove this polyfill once Symbol.dispose and Symbol.asyncDispose are available in V8.
   // eslint-disable-next-line node-core/prefer-primordials
-  Symbol.dispose ??= SymbolDispose;
+  if (typeof Symbol.dispose !== 'symbol') {
```

Comment:
We're guaranteed this runs before any userland code right? 

(Just making sure this is safe since it's weird we set up Symbol.dispose here (if someone tampered with symbol they can set up a getter that throws here) but we use ObjectDefineProperty which is a primordial that guards against this sort of tampering)

Reply:
I’m assuming that the file called “pre execution” runs before any userland code - if not, this polyfill is highly problematic conceptually :-)